services-js/commissions-search/src/App.js

```diff
@@ -1,10 +1,12 @@
 /* eslint-disable no-unused-vars */
 import React from 'react';
 import update from 'immutability-helper';
-
 import Search from './Search';
 import FacetList from './FacetList';
 import ResultList from './ResultList';
+import ReactGA from 'react-ga';
+
+ReactGA.initialize('UA-2187282-27');
```

Comment:
It’s not good to hard-code values like this into source code.

In theory configuration values like this should be made available by something in our Drupal template. See how we're doing that for the API key here: https://github.com/CityOfBoston/digital/blob/develop/services-js/commissions-search/src/index.js#L25

Though since you seem to want to use the same GA property as Boston.gov already has, do you really need `ReactGA`, as opposed to just assuming that the Google Analytics stuff is already loaded?

I'm worried that this will lead to double-counting.